Framework/Geometry/inc/MantidGeometry/Crystal/PeakShape.h

```diff
@@ -46,6 +47,10 @@ class DLLExport PeakShape {
   virtual int algorithmVersion() const = 0;
   /// Shape name
   virtual std::string shapeName() const = 0;
+  /// For selecting different radius types.
+  enum RadiusType { Radius = 0, OuterRadius = 1, InnerRadius = 2 };
```

Comment:
Initially I stared with an enum class, but quickly reverted back due to ParaView's lack of support for them. I think strongly typed enums would work if, as you mention above, we one uses an adapter to maintain the C++03 interface.